package.json

```diff
@@ -29,6 +29,9 @@
 	},
 	"main": "index.js",
 	"scripts": {
+		"test": "node index.js -d --tags @e2eCore @extendedTest",
+		"test:core": "node index.js -d --tags @e2eCore",
+		"test:extended": "node index.js -d --tags @extendedTest",
 		"test": "node index.js -d --tags @stableTest",
 		"test:core": "node index.js -d --tags @e2eCore",
 		"test:unstable": "node index.js -d --tags @unstableTest",
```

Comment:
the merge conflict was resolved in the wrong way here? the added lines are wrong and should be removed